### PR TITLE
[sailfish-browser] Removed obsoletes and added new sites. Fixes JB#51955

### PR DESCRIPTION
**Pull request by Nrsisti**

### CODE REVIEW

**Comment by rainemak** [comment]
Target to the next branch and rebase.

File: tests/manual/sites.txt [hunk 1]
```diff
@@ -48,49 +41,46 @@ instagram.com
...
 mywebsearch.com
 netflix.com
+openstreetmap.org
 osnews.com
 PayPal.com
-pcadvisor.co.uk
 phonearena.com
 pinterest.com
 pocketnow.com
+pornhub.com
```
Comment (rainemak) [inline tests/manual/sites.txt:56]: Please remove this.

File: tests/manual/sites.txt [hunk 2]
```diff
@@ -48,49 +41,46 @@ instagram.com
...
 together.jolla.com
 tomshardware.com
 trustedreviews.com
 tumblr.com
+twitch.tv
 twitter.com
 vk.com
-weibo.com
-wiki.merproject.org
+whatsapp.com
```
Comment (rainemak) [inline tests/manual/sites.txt:77]: No supported. Remove.

File: tests/manual/sites.txt [hunk 3]
```diff
@@ -48,49 +41,46 @@ instagram.com
 jollatides.com
 jollausers.com
 linkedin.com
-live.com
 livedoor.com
-m.txt.ch
```
Comment (rainemak) [inline tests/manual/sites.txt:53]: Why you removed this? This was working and used for testing numeric keyboard input.

File: tests/manual/fi-sites.txt [hunk 4]
```diff
@@ -35,25 +24,17 @@ golf.fi
 golfpiste.com
 hel.fi
 helsinki.fi
-helsinginsanomat.fi
+hs.fi
 hintaseuranta.fi
 historianet.fi
-hufvudstadsbladet.fi
```
Comment (rainemak) [inline tests/manual/fi-sites.txt:41]: hbl.fi

File: tests/manual/fi-sites.txt [hunk 5]
```diff
@@ -35,25 +24,17 @@ golf.fi
...
-indiedays.fi
-ilta-sanomat.fi
-jokakoti.fi
-juniori.fi
+is.fi
 kaksplus.fi
 kalamies.com
 kaleva.fi
 kauppalehti.fi
-keskisuomalainen.fi
```
Comment (rainemak) [inline tests/manual/fi-sites.txt:56]: ksml.fi

File: tests/manual/sites.txt (hunk 3 above)
Comment (Nrsisti) [reply tests/manual/sites.txt:53]: Added back

File: tests/manual/fi-sites.txt (hunk 4 above)
Comment (Nrsisti) [reply tests/manual/fi-sites.txt:41]: changed

File: tests/manual/fi-sites.txt (hunk 5 above)
Comment (Nrsisti) [reply tests/manual/fi-sites.txt:56]: changed

File: tests/manual/sites.txt (hunk 2 above)
Comment (Nrsisti) [reply tests/manual/sites.txt]: removed

File: tests/manual/sites.txt (hunk 1 above)
Comment (Nrsisti) [reply tests/manual/sites.txt]: removed

**Review by rainemak — APPROVED** [review]

**Pull request merged**